tests/python/gaia-ui-tests/gaiatest/apps/marketplace/app.py

```diff
@@ -70,6 +78,15 @@ def tap_install_button(self):
         self.root_element.find_element(*self._install_button_locator).tap()
         self.marionette.switch_to_frame()
 
+    def tap_open_app_button(self, app_title, _app_locator):
+        button = self.root_element.find_element(*self._install_button_locator)
+        Wait(self.marionette).until(lambda m: button.text == 'Open app')
```

Comment:
yup, I don't think I saw [data-l10n-text] value for this element, so I had to check for the text, I'll double check though.  If there isn't one, then I think we should make this test @parameterized for each locale.